### ISSUE TITLE
Fix 'Save revision to' for 'Working copy'

### ISSUE THREAD

**Issue by GoogleCodeExporter** [guberm/tortoisegit#475]
```
What steps will reproduce the problem?
1. Right click on a GIT repository
2. Show Log
3. Choose Working Copy
4. Choose a modified file
5. Right Click
6. Select 'Save revision to'
7. Add a filename in the filedialog and OK

What is the expected output?

It should save the given file to the desired folder under the desired name

What do you see instead?

A popup TortoiseGIT dialog with no message just one button.

So basically I don't know what happened and why and what I should do.


Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `lpapp.b...@gmail.com` on 17 Jun 2010 at 9:01
